### ISSUE TITLE
ETA Obi not activating

### ISSUE THREAD

**Issue by M1ntyPunch** [Sandrem/FlyCasual#2979]
This is on android.

Firstly, there were many times where Obi should be popping up as a trigger and he does not. The only thing I (not a programmer) can think of is that it may be counting himself in the ally count, which it shouldn't, but it would explain most of the occurrences. Though there were still some times where he was near (Range 1) enemies by himself and he still didn't trigger.
Secondly, even when the trigger does work, I go to activate it, and it does nothing.

Thank you for all the hard work you put into this; I hope this helps in fixing something.

**Comment by Sandrem** [comment]
Please, next time provide screenshot - it can be very useful to understand situation. (Maybe range is wrong, or all force tokens are spent?)
I checked code of this card, it ignores own ship during counting of ships for ability.
Try to reproduce this bug in hot seat mode. If you will have steps to reproduce bug with 100% chance - please, inform me.

**Comment by M1ntyPunch** [comment]
https://linksharing.samsungcloud.com/9KgI5OH3v9YL

Had to make a Samsung share link because the pics were too large.

I bring these screenshots significantly more confused than before.

I just dropped the other ship of the anakin/obi build and started testing
with stationary enemies in hotseat. It started as before, offering the
ability with no effect. Until I did a red maneuver and it worked great,
then the enemy pod fused into the bomber. After losing the stress, it
resumed not working.
